radicle-httpd/src/api.rs

```diff
@@ -1,5 +1,4 @@
-#[cfg(test)]
-pub mod test;
+pub mod auth;
```

Comment:
notice

Reply:
can do `cfg(test)` and `cfg(not(test))` here, does having non-public modules affect performance in any way?